common/views/components/Body/Body.js

```diff
@@ -124,10 +124,22 @@ const Body = ({ body, isDropCapped, pageId }: Props) => {
             )}
             {slice.type === 'contentList' && (
               <Layout8>
-                <SearchResults
-                  title={slice.value.title}
-                  items={slice.value.items}
-                />
+                {/* FIXME: this makes what-we-do and visit-us contentLists synchronous,
```

Comment:
I don't think this is a good idea, but putting it here if it turns out we're not serving core content to certain browsers.